crates/kernel/src/tasks/task.rs

```diff
@@ -273,24 +222,24 @@ impl Task {
                 trace!(?verb, ?player, ?vloc, ?args, "Starting verb");
 
                 let verb_call = VerbCall {
-                    verb_name: verb,
-                    location: vloc,
-                    this: vloc,
-                    player,
-                    args,
-                    argstr,
+                    verb_name: verb.clone(),
```

Comment:
I can't see a(n easy) way around these `.clone()`s, but: maybe make 'em `Cow`s?

Reply:
(Insert meme about `.clone()`-police)

Reply:
Yeah I think there's room throughout the code base to deal with some of the excessive string copies.

Things like verb names should actually probably be interned symbols, and we should consider moving towards that. (aisde: In ColdMUD's ColdC symbols were  a primitive type like Lisp, and I wish it was that way in MOO)